examples/applications/plot_cyclical_feature_engineering.py

```diff
@@ -227,8 +227,8 @@ def evaluate(model, X, y, cv):
         cv=cv,
         scoring=["neg_mean_absolute_error", "neg_root_mean_squared_error"],
     )
-    mae = -cv_results["test_neg_mean_absolute_error"]
-    rmse = -cv_results["test_neg_root_mean_squared_error"]
+    ##mae = cv_results["test_neg_mean_absolute_error"]
+    rmse = cv_results["test_neg_root_mean_squared_error"]
```

Comment:
We should keep the minus in front of `cv_results`.
The same thing for the previous line.